mathics/main.py

```diff
@@ -44,38 +44,7 @@ def __init__(self, definitions, colors, want_readline, want_completion):
         except ImportError:
             pass
 
-        # Try importing colorama to escape ansi sequences for cross platform
-        # colors
-        try:
-            from colorama import init as colorama_init
-        except ImportError:
-            colors = "NoColor"
-        else:
-            colorama_init()
-            if colors is None:
-                terminal_supports_color = (
-                    sys.stdout.isatty() and os.getenv("TERM") != "dumb"
-                )
-                colors = "Linux" if terminal_supports_color else "NoColor"
-
-        color_schemes = {
-            "NOCOLOR": (["", "", "", ""], ["", "", "", ""]),
-            "LINUX": (
-                ["\033[32m", "\033[1m", "\033[22m", "\033[39m"],
-                ["\033[31m", "\033[1m", "\033[22m", "\033[39m"],
-            ),
-            "LIGHTBG": (
-                ["\033[34m", "\033[1m", "\033[22m", "\033[39m"],
-                ["\033[31m", "\033[1m", "\033[22m", "\033[39m"],
-            ),
-        }
-
-        # Handle any case by using .upper()
-        term_colors = color_schemes.get(colors.upper())
-        if term_colors is None:
-            out_msg = "The 'colors' argument must be {0} or None"
-            print(out_msg.format(repr(list(color_schemes.keys()))))
-            quit()
+        term_colors = (["", "", "", ""], ["", "", "", ""])
```

Comment:
Won't this make the output non-coloured?

Reply:
Indeed, this make the CLI output non-coloured. I'm not sure this is a good idea...

Reply:
I mean, the only thing that was coloured was the `In[n]:=`/`Out[n]=` text, so it doesn't make much of difference in that regard. I'd just like to get some input from other people in @mathics/maintainers before approving this.

Reply:
If we were to remove the colours from the output we'd have to examine the code more carefully to remove the references to `term_colors`. There's no point in storing empty strings.

Reply:
Ths is safe in that in Mathics 1.0 colorama was optional. 

At some point I turned the _optional_ require into a madatory one.  

If this is less controversional will will cause less concern, I'd say just go back to the 1.0 code: https://github.com/mathics/Mathics/blob/v1.0/mathics/main.py and remove the required dependency in setup.py that I added subsequently.

Reply:
> It is safe. In Mathics 1.0 colorama was optional.
> 
> At some point I turned the _optional_ require into a madatory one, adding it to `setup.py`.
> 
> If this is less controversional will will cause less concern, I'd say just go back to the 1.0 code: https://github.com/mathics/Mathics/blob/v1.0/mathics/main.py and remove the required dependency in setup.py that I added subsequently.

Ohh I see. So your idea is to make it optional again? In that case, we're done, we can merge the PR. I thought we were ditching colours altogether in the CLI.

Reply:
At first I was going to do that. But since there is resistance, I'm going with what is in 1.0 which makes this optional. 

Here, you'd just pull in https://github.com/mathics/Mathics/blob/v1.0/mathics/main.py

Reply:
> At first I was going to ditch the colors altogether since it is only a half-hearted effort. A better job is done in mathicsscript

I'm actually quite in favour of ditching colorama altogether. My points are:

* I'd like to get some input from other maintainers before committing to this.
* If we are going to ditch colorama, there's no reason to keep the `term_colors` variable around (it's just a constant array of empty strings that is never used for anything).

Reply:
@rocky I can work on the second point if we decide to ditch colorama.

Reply:
> At first I was going to ditch the colors altogether since it is only a half-hearted effort.

Indeed, it doesn't get used much. That's why I'm leaning towards ditching it.

Reply:
Getting input from others is great! 

At this point, I don't have much of a vested interest one way or another. So please take this and finish it however you feel best. 

But some points to note, I want to minimize the effort on the CLI here, although have something basic as a fallback is nice. 

Over time other front-ends will be coming into place. Following the pattern of IMathics and iwolfram, they should be separate installable packages that manage their own dependencies, and have their own README's  and documentation. 

After 1.1.0, in mathicsscript, as it is now called I would to see/add support for RsT for documentation for the help system, support for mathplot/sympy graphics both window based and ASCII plots as a fallback (which exists in sympy) and so on.